storage/spanstore/downsampling_writer_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanstore
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+type noopWriteSpanStore struct{}
+
+func (n *noopWriteSpanStore) WriteSpan(span *model.Span) error {
+	return nil
+}
+func TestDownSamplingWriter_WriteSpan(t *testing.T) {
```

Comment:
this test is not testing anything, it only executes the writer. It should be checking that (a) span was or was not passed to the underlying writer, and (b) that the right metric was incremented.